plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java

```diff
@@ -4272,7 +4365,7 @@ public void registerCoreMembers() {
         // @Video /denizen/vids/Loops
         //
         // @Description
-        // TODO: Document Command Details
+        // Loops the commands within the braces so long as the tag returns true.

```

Comment:
Best to just copy the `@Short` description, then explain what *other* parts of this command do (stop/next)